app/models/TraderProfile.scala

```diff
@@ -104,4 +104,23 @@ object TraderProfile {
       case Left(errors) => Left(errors)
     }
 
+  def validateNiphlsUpdate(
```

Comment:
why aren't we using build niphls for this (the other function in this file)

Reply:
We don't need to retrieve the profile for the page load